test/controllers/users_controller_test.rb

```diff
@@ -10,7 +10,7 @@ class UsersControllerTest < ActionDispatch::IntegrationTest
 
   describe 'create method' do
     it 'should raise if user is not admin' do
-      User::ROLE.reject { |role| role == :admin }.each do |role|
+      User.role.values.reject { |role| role == :admin }.each do |role|
```

Comment:
For your edification, basically the difference is this (since I looked it up).

Since we were using import MongoID::Enum (which is how you'd bring in a class), I am guessing slightly here that access to Role before was via class, and not actual field, making it like ok, I want the User class, and then the ::Roles class on it. Now instead you have an actual field that has behavior added to it, so it's User.roles (field access) instead of User::Roles (class access). I'm typing this out mainly so someone can come in and go "No Jeffry you are totally wrong", too.